.github/cd.yaml

```
@@ -0,0 +1,107 @@
+name: Deploy to NPM and GitHub Package Registry
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)